pkg/config/autodiscover/helm_info.go

```diff
@@ -0,0 +1,22 @@
+package autodiscover
+
+type HelmSetList struct {
+	Items []HelmChart `json:""`
+}
+type HelmChart struct {
+	Name      string `json:"name"`
+	Namespace string `json:"namespace"`
+	Chart     string `json:"chart"`
+}
+
+func GetClusterHelmCharts() *HelmSetList {
+	out := execCommandOutput("helm list -A -o json")
```

Comment:
The test case should be restricted to the namespaces under test. We should also add a way to skip some helm charts, for instance adding a entry in the tnf_config.yml to skip some helm charts names in all namespace under test.